HISTORY.md

```diff
@@ -2,6 +2,9 @@
 ## Unreleased
 ### Public API Change
 * Support dynamically change `stats_dump_period_sec` option via SetDBOptions().
+* DB::Get in place of std::string accepts PinnableSlice, which avoids the extra memcpy of value to std::string in most of cases.
+    * PinnableSlice releases the pinned resources that contain the value when it is destructed or when ::Reset() is called on it.
+    * The old API that accepts std::string is deprecated
```

Comment:
Can you clarify what does "deprecated" mean? In this code base, we previously use "deprecated" as something already not supported. It's arguable whether we are using this word in a correct way, but I don't want people to get confused.